colibre/registration.py

```diff
@@ -163,37 +159,322 @@ def register_dust(self, catalogue, aperture_sizes):
+            )
+            dust_mass_large_grain_cd = getattr(
+                catalogue.dust_masses,
+                f"cold_dense_large_grain_mass_{aperture_size}_kpc",
+            )
+            dust_mass_small_grain_cd = getattr(
+                catalogue.dust_masses,
+                f"cold_dense_small_grain_mass_{aperture_size}_kpc",
+            )
             # All dust mass
             dust_mass_total = dust_mass_graphite + dust_mass_silicates
+            dust_mass_total_hi = dust_mass_graphite_hi + dust_mass_silicates_hi
+            dust_mass_total_h2 = dust_mass_graphite_h2 + dust_mass_silicates_h2
+            dust_mass_total_cd = dust_mass_graphite_cd + dust_mass_silicates_cd
+            dust_mass_total_neutral = dust_mass_graphite + dust_mass_silicates
+            small_to_large = dust_mass_small_grain / dust_mass_large_grain
+            small_to_large_h2 = dust_mass_small_grain_h2 / dust_mass_large_grain_h2
+            small_to_large_cd = dust_mass_small_grain_cd / dust_mass_large_grain_cd
 
         # In case run without dust
         except AttributeError:
             dust_mass_total = unyt.unyt_array(np.zeros_like(metal_frac), units="Msun")
 
-        # Fetch gas mass
+        dust_mass_total_hi.name = f"$M_{{\\rm dust,HI}}$ ({aperture_size} kpc)"
+        dust_mass_total_h2.name = f"$M_{{\\rm dust,H2}}$ ({aperture_size} kpc)"
+        dust_mass_total_cd.name = f"$M_{{\\rm dust,CD}}$ ({aperture_size} kpc)"
+        dust_mass_total_neutral.name = f"$M_{{\\rm dust,Neut.}}$ ({aperture_size} kpc)"
+
+        small_to_large_h2.name = f"$\\mathcal{{D}}_{{\\rm S}} / \\mathcal{{D}}_{{\\rm L}}$ ({aperture_size} kpc)"
+        small_to_large_cd.name = f"$\\mathcal{{D}}_{{\\rm S,CD}} / \\mathcal{{D}}_{{\\rm L,CD}}$ ({aperture_size} kpc)"
+        small_to_large.name = f"$\\mathcal{{D}}_{{\\rm S,Neut.}} / \\mathcal{{D}}_{{\\rm L,Neut.}}$ ({aperture_size} kpc)"
+
+        # Fetch gas masses
         gas_mass = getattr(catalogue.apertures, f"mass_gas_{aperture_size}_kpc")
+        atomic_mass = getattr(
+            catalogue.gas_hydrogen_species_masses, f"HI_mass_{aperture_size}_kpc"
+        )
+        molecular_mass = getattr(
+            catalogue.gas_hydrogen_species_masses, f"H2_mass_{aperture_size}_kpc"
+        )
+        neutral_mass = atomic_mass + molecular_mass
+        colddense_mass = getattr(
+            catalogue.cold_dense_gas_properties,
+            f"cold_dense_gas_mass_{aperture_size}_kpc",
+        )
+
+        # Metal mass fractions of the gas
+
+        linOH_abundance_times_mgas = getattr(
+            catalogue.lin_element_ratios_times_masses,
+            f"lin_O_over_H_total_times_gas_mass_100_kpc",
+        )
+
+        logOH_abundance_times_mhi = (
+            getattr(
+                catalogue.log_element_ratios_times_masses,
+                f"log_O_over_H_atomic_times_gas_mass_lowfloor_{aperture_size}_kpc",
+            )
+            / 0.7
```

Comment:
Could you add a comment that explains the origin of the `0.7` factor?

Reply:
thanks, there's not a good reason for these, think this is a hangover from debugging